src/exchange/group/accounts/group-accounts.controller.js

```diff
@@ -79,17 +92,53 @@ angular.module('Module.exchange.controllers').controller(
       }
     }
 
+    getDefaultDomain() {
+      const name = _.last(this.currentAccount.split('@'))
```

Comment:
You will ended up with an ESLint `semi` error.
```suggestion
      const name = _.last(this.currentAccount.split('@'));
```

Reply:
fixed